settings.py

```diff
@@ -0,0 +1,39 @@
+from dotenv import dotenv_values
+from decouple import config
+import pprint
+import os
+
+## Control switch for production and dev configurations
+# User specifies custom env var set to use on startup, else uses prod environment file
+envpath = os.environ.get('ENV_PATH', default='.env.remote')
+
+# Merge env vars from selected .env to the current environment (respecting unix way):
+os.environ.update(dotenv_values(env_path))
+
+## Settings class to store decoupled instance and build settings:
+class Settings:
+    def __init__(self):
+        # Knowledge settings
+        self.templates = config('templates', cast=str)
```

Comment:
Removed defaults, should fail if .env.remote is not being reached to respect principle of least surprise